target-platform/config/kura.target-platform.build.properties

```diff
@@ -28,6 +28,8 @@ jdk.dio.version=1.0.6-SNAPSHOT
 org.junit.version=4.10.0.v4_10_0_v20120426-0900
 org.apache.commons.fileupload.version=1.2.2.v20111214-1400
 org.apache.commons.io.version=2.4.0
+commons-exec.version=1.3
```

Comment:
I would collapse commons-exec.version and org.apache.commons.exec.version together referring as org.apache.commons.exec.version=1.3
I think that that duplication can be potentially misleading